html_css/intermediate_html/svgs.md

```diff
@@ -91,7 +91,7 @@ This section contains helpful links to other content. It isn’t required, so co
 3. If you want to get started making your own SVGs, you'll want some sort of visual editor.
    1. [This is a great little SVG editor](https://yqnn.github.io/svg-path-editor) for learning how the SVG markup works. It not only shows the XML, but also lets you see the "commands" you're using to generate the SVG. This website isn't really designed for making complex graphics, though.
    2. For that, an excellent option is [Inkscape](https://inkscape.org/), which is not only free but open source!
-   3. And if you're really invested in creating your own SVGs, you might want to check out some of the powerful paid options out there, such as [Affinity Designer](https://affinity.serif.com/designer/).
+   3. And if you're really interested in creating your own SVGs, you might want to check out some of the powerful paid options out there, such as [Affinity Designer](https://affinity.serif.com/designer/).
```

Comment:
Interested is the suitable word here because the sentence is based on the condition you have the desired to explore more on creating your own svgs rather than committing resources.

Reply:
In my opinion, invested works here just fine. A learner would actually **have to** commit resources in order to use Affinity Designer as it's a paid program. Committing financial resources towards your desire to create SVGs is more than just a result of interest and shows an investment towards the craft.